tests/monotouch-test/CoreBluetooth/CentralManagerTest.cs

```diff
@@ -78,7 +78,7 @@ public void SetUp ()
 		{
 			if (TestRuntime.IsInCI && TestRuntime.CheckXcodeVersion (14, 0))
 				TestRuntime.AssertNotDesktop (); // Looks like this particular test doesn't like Desktop + M1 bot machines
-			// iOS 13 and friends require bluetooth permission
+												 // iOS 13 and friends require bluetooth permission
```

Comment:
@rolfbjarne ein?

Reply:
It was added here: #17349, and autoformat disabled, so the change shows up in the next PR (yours)